tangelo/linq/tests/test_simulator.py

```diff
@@ -139,6 +139,25 @@ def test_simulate_mixed_state_save_measures(self):
             assert_freq_dict_almost_equal(sim.all_frequencies, reference_all, 8e-2)
             assert_freq_dict_almost_equal(sim.mid_circuit_meas_freqs, reference_mid, 8e-2)
 
+    def test_simulate_mixed_state_desired_state(self):
+        """ Test mid-circuit measurement (mixed-state simulation) for all installed backends.
+        Mixed-states do not have a statevector representation, as they are statistical mixtures of several quantum states.
```

Comment:
Mixed states.

Is this information relevant to the test ? Mixed states have a density matrix representation I believe, but I also am not sure why we'd mention any of that here.

Reply:
ok